addons/project_issue/migrations/7.0.1.0/post-migration.py

```diff
@@ -19,7 +19,31 @@
 #
 ##############################################################################
 from openupgrade import openupgrade
-from openerp import pooler
+from openerp import pooler, SUPERUSER_ID
+
+
+def migrate_categories(cr, pool):
+    category_ids = pool['crm.case.categ'].search(
+        cr, SUPERUSER_ID, [('object_id.model', '=', 'project.issue')])
+    crm_category2project_category = {}
+    for category in pool['crm.case.categ'].browse(
+            cr, SUPERUSER_ID, category_ids):
+        new_category_id = pool['project.category'].create(
+            cr, SUPERUSER_ID, {
+                'name': category.name,
+            })
+        crm_category2project_category[category.id] = new_category_id
+
+    for category_id, new_category_id in crm_category2project_category\
+            .iteritems():
+        cr.execute(
+            'insert into %s (%s, %s) '
+            'select id, %%s from project_issue where categ_id=%%s' %
+            pool['project.issue']._columns['categ_ids']._sql_names(
+                pool['project.issue']),
+            (new_category_id, category_id,))
+
+    pool['crm.case.categ'].unlink(cr, SUPERUSER_ID, category_ids)
```

Comment:
Please don't clean up these resources, even if they are unused in the core outside of this module. You don't know if there is any custom module that builds upon this functionality and still needs to perform its own migration at this point in the process.